cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -199,6 +207,33 @@ func uploadToS3(a *awscloud.AWS, outputDirectory, exportPath, bucket, key, filen
 	return url, nil
 }
 
+func (impl *OSBuildJobImpl) getContainerClient(destination string, targetOptions *target.ContainerTargetOptions) (*container.Client, error) {
+	destination, appliedDefaults := container.ApplyDefaultDomainPath(destination, impl.ContainersConfig.Domain, impl.ContainersConfig.PathPrefix)
+	client, err := container.NewClient(destination)
+	if err != nil {
+		return nil, err
+	}
+
+	if impl.ContainersConfig.AuthFilePath != "" {
+		client.SetAuthFilePath(impl.ContainersConfig.AuthFilePath)
+	}
+
+	if appliedDefaults {
+
```

Comment:
stray newline

Reply:
This was actually intentional (I like to let my code breath). But, np, I'll fix it. Just don't want to push such a minor change to kickoff all the CI for it. I'll do it as part of any other change that we might do

Reply:
Done